src/backend/defs/rest-api/objects/MessageV2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+type: object
+properties:
+  attachments:
+    type: array
+    items:
+      "$ref": Attachment.json
+  body: # in future, could be only a preview of body, limited to xxx chars
```

Comment:
![message_schema](https://cloud.githubusercontent.com/assets/65737/23998736/4d458a94-0a56-11e7-8742-9f49b1a7ca93.png)

It's more about class design, but the data model must permit it